scripts/publish/aws-s3.sh

```diff
@@ -31,20 +31,23 @@ function usage() {
   echo "    -b <s3 bucket>"
   echo "    -v <version>"
   echo "    -p <product name>"
+  echo "    -d include date in S3 key"
```

Comment:
I wonder if it'd be more flexible to actually make this `-d <string>  include string in S3 key` (and have it be optional), and then explicitly pass in the date string from higher-up-the-stack, as that'd then allow us to re-use the same functionality for publishing debug builds with custom keys?